### PR TITLE
add QCTools v0.7.3.20161011

### PR DESCRIPTION
**Pull request by dericed**

### CODE REVIEW

**Comment by vitorgalvao** [comment]
There was a [pull request template](https://github.com/blog/2111-issue-and-pull-request-templates) when you submitted. It must be filled.

If you were not presented a template, you may have submitted via a CLI tool (likely [`hub`](https://hub.github.com/)). It that was the case, you may wish to see if the tool has a way to let you use the template (`hub` [is working on it](https://github.com/github/hub/issues/1118)).

In either case, please [refer to the template](https://raw.githubusercontent.com/caskroom/homebrew-cask/master/.github/PULL_REQUEST_TEMPLATE.md) and update your comment.

File: Casks/qctools.rb [hunk 1]
```diff
@@ -0,0 +1,11 @@
+cask 'qctools' do
+  version 'v0.7.3.20161011'
+  sha256 '7a515466f541c07d37be94fbdeaca2669351e01a78ecb433b6cdf750b53a9342'
+
+  url 'https://github.com/bavc/qctools/releases/download/v0.7.3/QCTools_0.7.3.20161011_mac.dmg'
+  name 'QCTools'
+  homepage 'https://bavc.org/preserve-media/preservation-tools'
+  license :gratis
```
Comment (vitorgalvao) [inline Casks/qctools.rb]: Remove `license`, it has been deprecated and is no longer in use.

File: Casks/qctools.rb [hunk 2]
```diff
@@ -0,0 +1,11 @@
+cask 'qctools' do
+  version 'v0.7.3.20161011'
+  sha256 '7a515466f541c07d37be94fbdeaca2669351e01a78ecb433b6cdf750b53a9342'
+
+  url 'https://github.com/bavc/qctools/releases/download/v0.7.3/QCTools_0.7.3.20161011_mac.dmg'
```
Comment (vitorgalvao) [inline Casks/qctools.rb]: `url "https://github.com/bavc/qctools/releases/download/v#{version.major_minor_patch}/QCTools_#{version}_mac.dmg"`

File: Casks/qctools.rb [hunk 3]
```diff
@@ -0,0 +1,11 @@
+cask 'qctools' do
+  version 'v0.7.3.20161011'
```
Comment (vitorgalvao) [inline Casks/qctools.rb]: `version '0.7.3.20161011'`.

**Review by vitorgalvao — CHANGES_REQUESTED** [review]

**Comment by dericed** [comment]
Thanks @vitorgalvao, PR is updated with your suggestions.

File: Casks/qctools.rb [hunk 4]
```diff
@@ -0,0 +1,10 @@
+cask 'qctools' do
+  version '0.7.3.20161011'
+  sha256 '7a515466f541c07d37be94fbdeaca2669351e01a78ecb433b6cdf750b53a9342'
+
+  url "https://github.com/bavc/qctools/releases/download/v#{version.major_minor_patch}/QCTools_#{version}_mac.dmg"
```
Comment (vitorgalvao) [inline Casks/qctools.rb]: Please add an [`appcast`](https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/appcast.md).

(Was going to mention it before, but forgot)

**Review by vitorgalvao — CHANGES_REQUESTED** [review]

**Comment by dericed** [comment]
appcast data added, thx

**Review by vitorgalvao — APPROVED** [review]

**Pull request merged**